docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -0,0 +1,265 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.ML;
+using Microsoft.ML.Transforms;
+using static Microsoft.ML.DataOperationsCatalog;
+using System.Linq;
+using Microsoft.ML.Data;
+using System.IO.Compression;
+using System.Threading;
+using System.Net;
+
+namespace Samples.Dynamic
+{
+    public class ResnetV2101TransferLearningTrainTestSplit
+    {
+        public static void Example()
+        {
+            string assetsRelativePath = @"../../../assets";
+            string assetsPath = GetAbsolutePath(assetsRelativePath);
+
+            var outputMlNetModelFilePath = Path.Combine(assetsPath, "outputs", "imageClassifier.zip");
+
+            string imagesDownloadFolderPath = Path.Combine(assetsPath, "inputs", "images");
+
+            //Download the image set and unzip
+            string finalImagesFolderName = DownloadImageSet(imagesDownloadFolderPath);
+            string fullImagesetFolderPath = Path.Combine(imagesDownloadFolderPath, finalImagesFolderName);
+
+            try
+            {
+
+                MLContext mlContext = new MLContext(seed: 1);
+
+                //Load all the original images info
+                IEnumerable<ImageData> images = LoadImagesFromDirectory(folder: fullImagesetFolderPath, useFolderNameasLabel: true);
+                IDataView shuffledFullImagesDataset = mlContext.Data.ShuffleRows(mlContext.Data.LoadFromEnumerable(images));
+                shuffledFullImagesDataset = mlContext.Transforms.Conversion.MapValueToKey("Label")
+                    .Fit(shuffledFullImagesDataset)
```

Comment:
Let's keep it like this in any case. The only minor issue I see here is that if the user wants to remove the ValidationSet for the training pipeline and wants to simplify and do a single Fit() only for training, the changes in the code are more complex because he also has to realize that he needs to change the provided schema when saving the model. But it doesn't matter, I personally think you always will want to have the  ValidationSet for the training pipeline and see how it evolves while training... :)